### ISSUE TITLE
Find a way to extend the width of text displayed in the Monaco Editor autocompletion

### ISSUE THREAD

**Issue by anna-geller** [kestra-io/kestra#5823]
### Feature description

Some long plugin names are truncated, making it difficult to read. To reproduce, try to autocomplete the following:

```yaml
id: myflow
namespace: company.team
tasks:
  - id: hello
    type: io.kestra.plugin.core.log.Log
    message: Hello World! 

triggers:
  - id: flow
    type: io.kestra.plugin.core.trigger.Flow
    conditions:
      - id: flow
        type: ... # try autocomplete here
```
![image](https://github.com/user-attachments/assets/5f9eb9d2-a6ff-46bf-86c2-8d8efa43e49c)

This is a difficult issue as the width is hardcoded in Monaco editor code, so changing this would require us to manage a custom build:

![image](https://github.com/user-attachments/assets/b9e6b92b-80e1-47a6-90f2-1ecc67dab364)

**Note: this is not a good task for an external contributor.**

**Comment by brian-mulier-p** [comment]
The truth is even worst... The language server for yaml (the one used by monaco-yaml to provide autocompletion) [truncates the label to 57 characters](https://github.com/redhat-developer/yaml-language-server/blob/main/src/languageservice/services/yamlCompletion.ts#L239) so even though I managed to have a larger window it's still truncated at 57 characters...

Moreover due to hacky way of doing so the slider for resizing is becoming buggy so I'm not sure this is the way. Still I also found out that on top of the above language server limitation there is also a trim that is happening on monaco side (this one I'm able to get around through CSS as I said above).

So TLDR:
We could have larger window and get around some of the ellipsis truncate things, but some larger ones (DiscordIncomingWebhook) will still have ellipsis due to language server truncating before we get a hand on it

**Comment by anna-geller** [comment]
Brian will invesitgate a bit more, if not successful we'll close. This commit is already a big improvement: https://github.com/kestra-io/kestra/commit/cc083385f062fe692ab872c7b47739b5c487cb23
